src/overlays/actors/ovl_En_Elforg/z_en_elforg.c

```diff
@@ -440,7 +440,7 @@ void EnElforg_FreeFloating(EnElforg* this, PlayState* play) {
     if (!Player_InCsMode(&play->state)) {
         if ((this->actor.xzDistToPlayer < 30.0f) && (scaledYDistance < 12.0f) && (scaledYDistance > -68.0f)) {
             EnElforg_SetupFairyCollected(this, play);
-            func_80115908(play, 48);
+            Health_ChangeBy(play, 48);
```

Comment:
```suggestion
            Health_ChangeBy(play, 0x30);
```
Health works much better in hex since it's 0x10 to the heart.